docs/standard/analyzers/portability-analyzer.md

```diff
@@ -7,35 +7,66 @@ ms.assetid: 0375250f-5704-4993-a6d5-e21c499cea1e
+### Pick Target Platforms
+To begin using the .NET Portability Analyzer in Visual Studio, you first need to download and install the extension from the [Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=ConnieYau.NETPortabilityAnalyzer). It works on Visual Studio 2017 and later versions. You can configure it in Visual Studio via **Analyze** > **Portability Analyzer Settings** and select your Target Platforms, which is the .NET platforms/versions that you want to evaluate the portability gaps comparing with the platform/version that your current assembly is built with.
 
 ![Portability screenshot](./media/portability-analyzer/portability-screenshot.png)
 
-To analyze your entire project, right-click on your project in **Solution Explorer** and select **Analyze Assembly Portability**. Otherwise, go to the **Analyze** menu and select **Analyze Assembly Portability**. From there, select your project’s executable or DLL.
+To use the ApiPort console app, download it from [ApiPort repository](http://aka.ms/apiportdownload). It comes with both .NET Core version and .NET Framework version. You can  use "listTargets" command option to display the available target list, then pick target platforms by specifying -t or --target command option.
 
-![Portability Analyzer from Solution Explorer](./media/portability-analyzer/portability-solution-explorer.png)
-
-After running the analysis, you'll see your .NET Portability Report. Only types that are unsupported by a target platform appear in the list and you can review recommendations in the **Messages** tab in the **Error List**. You can also jump to problem areas directly from the **Messages** tab.
+### Analyze portability
+To analyze your entire project in Visual Studio, right-click on your project in **Solution Explorer** and select **Analyze Assembly Portability**. Otherwise, go to the **Analyze** menu and select **Analyze Assembly Portability**. From there, select your project’s executable or DLL.
 
-![Portability Report](./media/portability-analyzer/portability-report.png)
+![Portability Analyzer from Solution Explorer](./media/portability-analyzer/portability-solution-explorer.png)
 
-If you don’t want to use Visual Studio, you can use the Portability Analyzer from the command prompt. Just download the API Portability Analyzer from the [Microsoft/dotnet-apiport](https://github.com/Microsoft/dotnet-apiport/releases) repository.
+If you want to use [ApiPort console app](https://aka.ms/apiportdownload).
 
 * Type the following command to analyze the current directory: `\...\ApiPort.exe analyze -f .`
```

Comment:
Can you add something to the effect that you only want to submit user assemblies?

Reply:
Added some guidance about this